### PR TITLE
Use & concatenation.

### PR DESCRIPTION
**Pull request by matkoch**
Following up on #219. Unfortunately I couldn't test this locally.

Now it always appends a '?'. Then followed by the individual key value pairs separated by '&'.

### CODE REVIEW

File: src/server/controller/badge.js [hunk 1]
```diff
@@ -48,11 +48,11 @@ router.all('/readme/badge/:owner/:repo', function (req, res) {
     var colorB = req.query && req.query.colorB ? req.query.colorB : undefined;
     var logo = req.query && req.query.logo ? req.query.logo : undefined;
     var redirect = function (count) {
-        var url = 'https://img.shields.io/badge/CLAs signed-' + count + '-0594c6.svg';
-        url = style ? url + '?style=' + style : url;
-        url = label ? url + '?label=' + label : url;
-        url = colorB ? url + '?colorB=' + colorB : url;
-        url = logo ? url + '?logo=' + logo : url;
+        var url = 'https://img.shields.io/badge/CLAs signed-' + count + '-0594c6.svg?';
```
Comment (KharitonOff) [inline src/server/controller/badge.js]: If i don't use any of the parameters, the url becomes like "https://img.shields.io/badge/CLAs%20signed-39-0594c6.svg?"

Comment (matkoch) [inline src/server/controller/badge.js]: Simplicity vs. complexity :) The url is never visible to end-users.

Comment (KharitonOff) [inline src/server/controller/badge.js]: That would be ok, but this url doesn't work then ;)

Comment (matkoch) [inline src/server/controller/badge.js]: Sure it does :o Tested in chrome, firefox and internet explorer.

Comment (KharitonOff) [inline src/server/controller/badge.js]: oh, now it works... hm... ok, let's do it :)

Comment (matkoch) [inline src/server/controller/badge.js]: thanks :)

**Comment by KharitonOff** [comment]
Thank you again for your contribution 🙂 👍

**Pull request merged**